### PR TITLE
add version querying

### PR DESCRIPTION
**Pull request by steve-chavez**
Related to https://github.com/PostgREST/postgrest/issues/2647 and https://github.com/PostgREST/postgrest/issues/2848

### CODE REVIEW

File: docs/references/admin.rst [hunk 1]
```diff
@@ -83,12 +84,32 @@ Restart the database and watch the log file in real-time to understand how HTTP
 Server Version
 --------------
 
-When debugging a problem it's important to verify the PostgREST version. Look for the :code:`Server` HTTP response header, which contains the version number.
+When debugging a problem it's important to verify the running PostgREST version. There are 3 ways you can check the version:
```
Comment (laurenceisla) [inline docs/references/admin.rst:87]:
```suggestion
When debugging a problem it's important to verify the running PostgREST version. There are three ways to do this:
```

File: docs/references/admin.rst [hunk 2]
```diff
@@ -83,12 +84,32 @@ Restart the database and watch the log file in real-time to understand how HTTP
...
+
+- Look for the :code:`Server` HTTP response header, which contains the version number. This header will be sent on all the requests.
```
Comment (laurenceisla) [inline docs/references/admin.rst:89]:
```suggestion
- Look for the :code:`Server` HTTP response header that PostgREST returns on every request.
```

I think that "which contains the version number" is implied in the previous paragraph.

File: docs/references/admin.rst [hunk 3]
```diff
@@ -83,12 +84,32 @@ Restart the database and watch the log file in real-time to understand how HTTP
...
 
+  HEAD /users HTTP/1.1
+
   Server: postgrest/11.0.1
 
+- Query ``application_name`` on `pg_stat_activity <https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW>`_.
+
+.. code-block:: psql
+
+  select distinct application_name from pg_stat_activity where application_name ilike '%postgrest%';
```
Comment (laurenceisla) [inline docs/references/admin.rst:101]:
```suggestion
  select distinct application_name
  from pg_stat_activity
  where application_name ilike '%postgrest%';
```

File: docs/references/admin.rst (hunk 2 above)
Comment (steve-chavez) [reply docs/references/admin.rst:89]:
```suggestion
- Look for the :code:`Server` HTTP response header that is returned on every request.
```

**Review by laurenceisla — APPROVED** [review]

**Pull request merged**